test/Bench/ClassyBench.php

```diff
@@ -23,7 +23,16 @@
      */
     public function benchBaseline()
     {
-        \iterator_to_array(new File\ClassFileLocator($this->directory()));
```

Comment:
We aren't interested in the array of `Zend\File\PhpClassFile`, but the array of classes!